### PR TITLE
Created method to request daily water intake

### PR DESCRIPTION
**Pull request by jenspots**
## Description
This pull request changes two things:
- Firstly, a new method was added to request the water intake of a given day called `getWater`.
- Secondly, line 390 of `RCTAppleHealthKit/RCTAppleHealthKit+Methods_Dietary.m` was changed so `saveWater` accepts a proper instance of the `HealthValueOptions` interface. The key `water` was changed to `value`. This **will** break code that relies on this method.

Ofcourse the documentation was updated to reflect these changes.

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have checked my code and corrected any misspellings

## Disclaimer
I am not an Objective-C programmer and largely based these changes on existing code in this repository. I mainly forked this repository to develop a very small hydration app for iOS. If any part of this PR/code is not up to standards let me know and I'll change it accordingly.

### CODE REVIEW

File: docs/saveWater.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Example input options:
 
 ```javascript
 let options = {
-  water: 0.2 // Amount in liters
+  value: 0.2, // Amount in liters
```
Comment (lucaspbordignon) [inline docs/saveWater.md:12]: Great catch!

**Review by lucaspbordignon — APPROVED** [review]
Hey @jenspots !

Thanks a lot for the addition and also for the minor fixes around the codebase!

**Comment by AlbertoMeQ** [comment]
As an extension, is it possible to pass in a `start_date` and `end_date` to get a list of the water drank for each day? Otherwise I need to make several calls just to get 1 week.

**Comment by AlbertoMeQ** [comment]
P.S. @jenspots have you ever seen a fatal error when trying to call `getWater`?

**Pull request merged**